test/connectors/FileNotificationOrchestratorConnectorSpec.scala

```diff
@@ -62,13 +61,12 @@ class FileNotificationOrchestratorConnectorSpec extends SpecBase {
         Matchers.any()
       )(Matchers.any(),
         Matchers.any(),
-        argumentCaptorForHeaderCarrier.capture(),
```

Comment:
I think the 'argumentCaptorForHeaderCarrier' val can be removed further up in this file?